tests/integration/v3_grpc_test.go

```diff
@@ -1922,7 +1922,45 @@ func TestV3LargeRequests(t *testing.T) {
 					t.Errorf("#%d: range expected no error, got %v", i, err)
 				}
 			}
+		})
+	}
+}
 
+// TestV3AdditionalGRPCOptions ensures that configurable GRPCAdditionalServerOptions works as intended.
+func TestV3AdditionalGRPCOptions(t *testing.T) {
+	integration.BeforeTest(t)
+	tests := []struct {
+		maxRequestBytes uint
```

Comment:
Please add one more field `name` into the struct to clarify the purpose of the case.